extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -613,6 +614,9 @@ class CornerstoneViewportService extends PubSubService implements IViewportServi
    * case where the user is in MPR and a viewport other than active should be
    * the one to change to display the iamge.
    *
+   * Third choice is to use a viewport whose orientation best matches the
```

Comment:
Can you extract the three types into separate methods, each with their own comments?  That reduces the total complexity of the top level method and easier to understand what is going on.